src/Components/CreateImageWizard/steps/fileSystemConfiguration.js

```diff
@@ -111,7 +125,10 @@ const fileSystemConfigurationStep = {
         </TextContent>
       ),
       condition: {
-        or: [{ when: 'file-system-config-radio', is: 'automatic' }],
+        or: [
+          { when: 'file-system-config-radio', is: 'automatic' },
+          { when: 'oscap-policy', is: undefined },
+        ],
```

Comment:
It seems that if there are no fs customizations in the policy, the automatic partitioning text shows:

"xccdf_org.ssgproject.content_profile_anssi_bp28_minimal"
![Screenshot from 2023-10-06 10-55-37](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/1a6e74d2-918b-4037-9d2f-c63f34d6ae08)

Reply:
The automatic partitioning text seems to be there even if I leave the OpenSCAP Compliance set as Do not add a policy and then choose manual partitioning.

Reply:
I got this fixed.